Dockerfile.dapper

```diff
@@ -9,16 +9,18 @@ RUN apt-get update && \
     rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
-    GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
+    GOPATH=/go PATH=/go/bin:/usr/local/go/bin:/usr/bin/docker:${PATH} SHELL=/bin/bash
```

Comment:
Any reason for adding `usr/bin/docker` in the path? assuming /usr/bin is already in path by default

Reply:
Because it is required docker 1.12.* to build windows image.
As below, download docker 1.12.6 and Extract into dir `/usr/bin/docker`